templates/ohdsi-webapi/ohdsi_atlas_ui.bicep

```diff
@@ -0,0 +1,123 @@
+param location string
+param suffix string
+param appServicePlanId string
+param ohdsiWebApiUrl string
+
+
+var dockerRegistryServer = 'https://index.docker.io/v1'
+var dockerImageName = 'ohdsi/atlas'
+var dockerImageTag = 'latest'
+var shareName = 'atlas'
+var mountPath = '/etc/atlas'
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2021-04-01' = {
+  name: 'atlasui${suffix}'
```

Comment:
Can we use a generic name that will enable us to use this storage account for other things in the future?

Reply:
Done